src/main/java/edu/uc/cech/agrawadv/eventorganizer/resource/Resource.java

```diff
@@ -3,16 +3,19 @@
 import java.util.UUID;
 
 public class Resource {
+	
     private UUID resourceId;
     private String title;
     private String description;
 
+    //Didn't include ID in parameter so the resource is assigned with an ID
```

Comment:
Added this to show the reader why the ID is not included. It may be obvious but to some a little comment may help. ref: https://dzone.com/articles/10-tips-how-to-improve-the-readability-of-your-sof